tests/components/solarlog/snapshots/test_sensor.ambr

```diff
@@ -149,7 +347,7 @@
     'last_changed': <ANY>,
     'last_reported': <ANY>,
     'last_updated': <ANY>,
-    'state': '85.0',
+    'state': '0.85',
```

Comment:
Looks like one value got borked

Reply:
This is correct, conversion to percentage has been moved from component to library.

Reply:
But perhaps I should amend the fixture, to give more realistic values...